run_testlist.py

```diff
@@ -15,106 +15,220 @@
 env['DST_DIR']  = "<path_to_daos_scaled_testing>" #/scratch/TESTS/daos_scaled_testing
 env['RES_DIR']  = "<path_to_result_dir>" #/home1/06753/soychan/work/POC/TESTS/dst_framework/RESULTS
 
-slf_testlist = [ { 'testcase': 'st_1tomany_cli2srv_inf1',
-               'nServer' : [ 2,  4,  8, 16, 32, 64, 128, 256, 512],
-               'nClient' : [ 1,  1,  1,  1,  1,  1,   1,   1,   1],
-               'timeout' : [15, 15, 15, 15, 15, 15,  15,  15,  15], #timeout in minutes
-               'ppc'     : 1,
-               'inflight': 1,
-               'enabled' : 0
-             },
-             { 'testcase': 'st_1tomany_cli2srv_inf16',
-               'nServer' : [ 2,  4,  8, 16, 32, 64, 128, 256, 512],
-               'nClient' : [ 1,  1,  1,  1,  1,  1,   1,   1,   1],
-               'timeout' : [15, 15, 15, 15, 15, 15,  15,  15,  15], #timeout in minutes
-               'ppc'     : 1,
-               'inflight': 16,
-               'enabled' : 0
-             }
-            ]
-
-ior_testlist = [ { 'testcase': 'ioreasy_1to4',
-               'nServer' : [ 2,  4,  8, 16],
-               'nClient' : [ 8, 16, 32, 64],
-               'timeout' : [15, 15, 15, 15], #timeout in minutes
-               'ppc'     : 32,
-               'pool_sz' : '85G',
-               'xfer_sz' : '1M',
-               'blk_sz'  : '1G',
-               'enabled' : 0
-             },
-             { 'testcase': 'ioreasy_c16',
-               'nServer' : [ 2,  4,  8, 16],
-               'nClient' : [16, 16, 16, 16],
-               'timeout' : [15, 15, 15, 15], #timeout in minutes
-               'ppc'     : 32,
-               'pool_sz' : '85G',
-               'xfer_sz' : '1M',
-               'blk_sz'  : '1G',
-               'enabled' : 0
-             }
-            ]
+slf_testlist = [{'testcase': 'st_1tomany_cli2srv_inf1',
```

Comment:
I think we can also put mpi config here and source build_env, and remove line 47-48 in tests.sh.

Reply:
Done